qds_sdk/commands.py

```diff
@@ -254,6 +254,98 @@ def parse(cls, args):
         v["command_type"] = "HiveCommand"
         return v
 
+class SparkCommand(Command):
+	
+	#{"program":prog,"language":"python","command_type":"SparkCommand","label":"praveen-spark"} 
+    usage = ("sparkcmd <submit|run> [options]")
+    allowedlandlist = ["python", "scala"]
+    optparser = GentleOptionParser(usage=usage)
+    optparser.add_option("--program", dest="program", help="program code")
```

Comment:
Also, I think we should ask for a program file here instead of program code.
No one is going to type all the program code in the command line.
There is an example of this type of interaction - search for `custom_config_file`.